Framework/Algorithms/src/MonteCarloAbsorption.cpp

```diff
@@ -108,6 +448,26 @@ void MonteCarloAbsorption::init() {
 
   InterpolationOption interpolateOpt;
   declareProperty(interpolateOpt.property(), interpolateOpt.propertyDoc());
+  declareProperty("SparseInstrument", false, "Enable simulation on special "
+                                             "instrument with a sparse grid of "
+                                             "detectors interpolating the "
+                                             "results to the real instrument.");
+  auto twoOrMore = boost::make_shared<Kernel::BoundedValidator<int>>();
+  twoOrMore->setLower(2);
```

Comment:
Good point. The bound can be changed later to 2 if needed without breaking the API. I'll change the default to an odd number as well.